tests/platform_helper/providers/schema_migrations/test_schema_v0_to_v1_migration.py

```diff
@@ -0,0 +1,105 @@
+from dbt_platform_helper.providers.schema_migrations.schema_v0_to_v1_migration import (
+    SchemaV0ToV1Migration,
+)
+
+
+class TestSchemaV1ToV2Migration:
+    def test_from_version(self):
+        migration = SchemaV0ToV1Migration()
+
+        assert migration.from_version() == 0
+
+    def test_removes_terraform_platform_modules_version_from_defaults(self):
+        migration = SchemaV0ToV1Migration()
+        config = {
+            "default_versions": {"terraform-platform-modules": "5.1.0", "platform-helper": "14.0.0"}
+        }
+
+        actual_config = migration.migrate(config)
+
+        # Ensure the migration does not modify the original
+        assert actual_config != config
```

Comment:
For readability, might be better as...
```suggestion
        assert migrated_config != original_config
```

Reply:
I think you could extract this to a method (`assert_original_config_is_not_changed`) too instead of having the comment.